src/overlays/components/OverlayController.tsx

```diff
@@ -150,7 +154,7 @@ const OverlayProvider: FunctionComponent = props => {
 
   const closer = useCallback(() => {
     dispatch(dismissOverlay())
-    if (onClose) {
+    if (onClose && typeof onClose === 'function') {
       onClose()
```

Comment:
can onClose be anything else?

Reply:
yes it can

Reply:
Wait, why? How?

Reply:
still trying to figure that out, but somehow onClose becomes an action object. I figured that ill look into it, but in the meantime, if we are going to have a conditional that allows us to call a function, we should make sure its a function regardless

Reply:
ah found the issue, pushing up the fix now